cachi2/core/package_managers/rubygems.py

```diff
@@ -0,0 +1,336 @@
+import logging
```

Comment:
Please don't put everything into the same module - those are the old ways we did implement package managers leading to a single big messy module. We should split the logic into a few self-contained modules wrapped in a package.

Reply:
The file will be deleted anyway, so I did not bother with splitting it.